### ISSUE TITLE
Bug: v8.x Sprites not rendering in Chrome/Edge Windows 11

### ISSUE THREAD

**Issue by whitetigle** [pixijs/pixijs#10375]
### Current Behavior

[Sprites in pixi Playground](https://pixijs.com/8.x/playground?exampleId=basic.tinting) do not render with v8.x on Google Chrome and Microsoft Edge. 

However: 
- Graphics do render
- sprites do render in Firefox.

### Expected Behavior

[Samples in pixi Playground](https://pixijs.com/8.x/playground?exampleId=basic.tinting) should render with v8.x on Google Chrome and Microsoft Edge

### Steps to Reproduce

Load any example using sprites in Pixi Playground on windows. The screen will be black. 

### Environment

- **`pixi.js` version**: *v8.x*
- **Browser & Version**: * Chrome 123.0.6312.59*
- **OS & Version**: *Windows 11*
- **Running Example**: *<https://pixijs.com/8.x/playground?exampleId=basic.tinting/>*

**Comment by TheRDavid** [comment]
I have the same issue with Edge and Chrome, also on Opera. In the examples, for instance the "Animated Sprite Jet".
It does work when switching to v7, but not on v8.

**Comment by GoodBoyDigital** [comment]
thanks for reporting @whitetigle ! can i check - if you force the WebGL renderer does it render ok?

```
await app.init({ 
  resizeTo: window,
  preference:'webgl'
});
```

thanks!

**Comment by whitetigle** [comment]
> thanks for reporting @whitetigle ! can i check - if you force the WebGL renderer does it render ok?
> 
> ```
> await app.init({ 
>   resizeTo: window,
>   preference:'webgl'
> });
> ```
> 
> thanks!

Yes it works! 👍 (does it mean webgpu is not available on my chrome?)

**Comment by whitetigle** [comment]
Interestingly, I activated the `enable-unsafe-webgpu` flag, but it does not display anything.

**Comment by GoodBoyDigital** [comment]
thanks @whitetigle - this lends more credence to this PR #10367 .
it seems that WebGPU is not 100% stable across all things yet!
